cmd/notation/logout.go

```diff
@@ -40,16 +41,12 @@ func logoutCommand(opts *logoutOpts) *cobra.Command {
 func runLogout(ctx context.Context, opts *logoutOpts) error {
 	// set log level
 	ctx = opts.LoggingFlagOpts.SetLoggerLevel(ctx)
-
-	// initialize
-	serverAddress := opts.server
-	nativeStore, err := auth.GetCredentialsStore(ctx, serverAddress)
+	credsStore, err := auth.NewCredentialsStore()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to get credentials store: %v", err)
 	}
-	err = nativeStore.Erase(serverAddress)
-	if err != nil {
-		return err
+	if err := credentials.Logout(ctx, credsStore, opts.server); err != nil {
```

Comment:
What will happen if no native credential store?

Reply:
`Logout` is to delete the credential for the given server address. If there is no native credentials store, `oras-credentials-go` will try to delete the credentials from the config file. If the credential is not present in the config file, or the config file does not exist, the deletion will result in no operation.
See https://github.com/notaryproject/notation/pull/654#issuecomment-1525562629 for details.